Client/WebFilters/CosmeticFiltersResourceDownloader.swift

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 The Presearch Authors. All rights reserved.
+// Copyright 2022 The Brave Authors. All rights reserved.
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
lines 49 - 53 (for `endpoint`) have brave.com there still. Can you update those please? 😄